src/Codeception/Module/Yii2.php

```diff
@@ -55,6 +55,8 @@ public function _before(\Codeception\TestCase $test)
     {
         $this->client = new \Codeception\Lib\Connector\Yii2();
         $this->client->configFile = \Codeception\Configuration::projectDir().$this->config['configFile'];
+        $this->client->setServerParameter('HTTP_HOST', parse_url(\Codeception\Configuration::config()['config']['test_entry_url'], PHP_URL_HOST));
```

Comment:
Hmm... There are a lot of Yii2 code related to this parameter already e.g. https://github.com/yiisoft/yii2-app-advanced/search?q=test_entry_url
But your idea seems reasonable to users that have their applications installed before this paramater was introduced.